src/Native/System.Security.Cryptography.Native/pal_bio.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_bio.h"
+
+#include <assert.h>
+
+extern "C" BIO* BioCreateNew()
```

Comment:
I'd go with `CreateMemoryBio`, `BioNewMemory` or something; to indicate that it's already been assigned a BIO type.  (BIO is similar to Stream; this is effectively MemoryStream)